config/src/main/kotlin/org/springframework/security/config/annotation/web/AuthorizeHttpRequestsDsl.kt

```diff
@@ -222,6 +223,13 @@ class AuthorizeHttpRequestsDsl : AbstractRequestMatcherDsl() {
         return AuthorityAuthorizationManager.hasAnyRole(*roles)
     }
 
+    /**
+     * Require a specific IP or range of IP addresses.
+     * @since 6.1
```

Comment:
Will you please change this to `@since 6.3`? That is the version this would go out on.